Formula/pilosa.rb

```diff
@@ -15,13 +17,23 @@ class Pilosa < Formula
   depends_on "go" => :build
   depends_on "glide" => :build
 
+  go_resource "github.com/rakyll/statik" do
```

Comment:
Pilosa already uses `dep` for library dependencies (recently switched from Glide). However, `statik` in this case is a binary dependency and is used as part of the build process (via `go generate`). As far as I'm aware, it is not possible to install a binary from a vendored dependency.

Here is a relevant ticket in the Glide issue tracker: https://github.com/Masterminds/glide/issues/418

And here is an existing homebrew-core package that includes `statik` using the same method: https://github.com/Homebrew/homebrew-core/blob/master/Formula/aurora.rb